src/OpenTelemetry.Exporter.Jaeger/.publicApi/net46/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,22 @@
+OpenTelemetry.Exporter.Jaeger.JaegerExporter
+OpenTelemetry.Exporter.Jaeger.JaegerExporter.Process.get -> OpenTelemetry.Exporter.Jaeger.Process
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.AgentHost.get -> string
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.AgentHost.set -> void
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.AgentPort.get -> int
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.AgentPort.set -> void
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.JaegerExporterOptions() -> void
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.MaxPayloadSizeInBytes.get -> int?
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.MaxPayloadSizeInBytes.set -> void
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.ProcessTags.get -> System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, object>>
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.ProcessTags.set -> void
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.ServiceName.get -> string
+OpenTelemetry.Exporter.Jaeger.JaegerExporterOptions.ServiceName.set -> void
+OpenTelemetry.Exporter.Jaeger.Process
+OpenTelemetry.Exporter.Jaeger.Process.Process(string serviceName) -> void
+OpenTelemetry.Exporter.Jaeger.Process.Process(string serviceName, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, object>> processTags) -> void
+OpenTelemetry.Exporter.Jaeger.Process.ServiceName.get -> string
+OpenTelemetry.Trace.JaegerExporterHelperExtensions
+override OpenTelemetry.Exporter.Jaeger.JaegerExporter.Export(in OpenTelemetry.Batch<System.Diagnostics.Activity> activityBatch) -> OpenTelemetry.ExportResult
+override OpenTelemetry.Exporter.Jaeger.Process.ToString() -> string
```

Comment:
this might have ben made public by mistake. Not sure why this getter is needed on exporter

Reply:
fixed!